alloy/send-logs-to-loki/step2.md

```diff
@@ -74,4 +75,7 @@ This Docker Compose file includes Loki and Prometheus configured as data sources
     docker-compose up -d
    ```{{exec}}
 
+> If you encounter the following error when you start your Docker container, `docker: 'compose' is not a docker command`{{copy}}, use the command `docker-compose up`{{copy}} to start your Docker container.
+> <!-- raw HTML omitted -->
```

Comment:
I bet the pre-processed Markdown for admonitions within ignore directives looks funny: https://github.com/grafana/alloy/blob/main/docs/sources/tutorials/send-logs-to-loki.md?plain=1#L175-L179